arc/src/main.c

```diff
@@ -75,9 +78,12 @@ static void queue_message(struct zjs_ipm_message *incoming_msg)
     k_sem_give(&arc_sem);
 }
 
-static void ipm_msg_receive_callback(void *context, u32_t id, volatile void *data)
+static void ipm_msg_receive_callback(void *context,
+                                     u32_t id,
```

Comment:
To avoid over 80chars for some of these functions, clang actually changes it to,

static void
ipm_msg_receive_callback(void *context, u32_t id, volatile void *data)

If that's something you prefer?